stack/git/README.md

```diff
@@ -1,155 +1,140 @@
+You will need a [GitHub Account](https://github.com/join?source=header-home) and configure your  [ssh key with github](https://medium.com/codebase/configuring-ssh-key-for-github-9d2416a377ae) to collaborate with the IcaliaLabs git repositories.
 
-## Set up your laptop
 
-You can easily set up your laptop by running [kaishi](https://github.com/IcaliaLabs/kaishi), a shell script to convert any Mac OS X or Linux computer into a real development machine. This will install the latest version of git. 
+## 1.2. Basics
+If you never use git version control before you can  continue with this guide you should check this awesome resources to learn git:
 
-You can make a clean installation of git using the installation instructions from the official [website](https://git-scm.com/downloads).
+- [Git - the simple guide](http://rogerdudler.github.io/git-guide/)
+- [Learn Git Version Control using Interactive Browser-Based Scenarios](https://www.katacoda.com/courses/git)
 
-## Branch structure
+## 1.3. Icalia code flow
+We at Icalia Labs never work directly on the master branch to build software, we use a git workflow to keep things isolated.
 
-We at [Icalia Labs](http://icalialabs.com) never work directly on the `master` branch to build software, we like to keep things isolated. We even give only access to a developer or group of developers who are in charge of merging the code. Although we make a branch per feature, we consider an extra branch called `dev` where we mantain all of the source code HEAD, so `dev` always reflect the most recent changes to be released.
+Lets take a look to this workflow: 
 
-Once the `dev` branch is stable enough we then merge with `master` and tagged with a version number. This way we keep things sane, and the CI server can easily push to production or rollback to the last stable version on production.
+![](/stack/git/img/step_1.png)
 
-An image is presented below to demostrate this:
+### 1.3.1. Create a Branch from Master
 
-![http://nvie.com/posts/a-successful-git-branching-model](http://nvie.com/img/main-branches@2x.png)
+You’ll develop your new feature, knowing that when finished, your feature will be ready to be deployed, instead of waiting in “inventory”.
 
+- Keep small code features.
+- **You must keep track of changes coming from “master” while your code is still in development.**
+***
+![](/stack/git/img/step_2.png)
 
-## Writing a new feature
+### 1.3.2. Create Code, add Commits, Push to Github
+Automated CI tools (Drone, Foresight) will start giving you feedback throughout the process.
 
-Now that we understand how the `dev/master` flow works, it is time to get to know how to start a new feature.
+***
 
-First create a local feature branch off `dev`:
+![](/stack/git/img/step_3.png)
 
-```console
-$ git pull origin dev
-$ git checkout -b feat/<feature-branch>
-```
+### 1.3.3. Create a Pull Request at Github
```

Comment:
Maybe talk about more the PR template we use at the company?

Reply:
@kurenn  Something like this: 

Remember to use the Icalialabs PR template in order to provide important information abut your pull request.

This template consists in two sections:

- A list for describing  what the pull request does
- A list of checkboxes with information about the status of the pull request( tests status, branch status, code style issues)
 ```
	### What does this PR do?

  * Add "create post" button
  * etc

	#### Merge Checklist:

	- [ ] The branch is up-to-date (i.e. rebased) with the base branch
	- [ ] All tests are passing - see ["Running tests"](../#running-tests)
	- [ ] No code style issues
 ```

Reply:
Yeah that works!